### DISCUSSION TITLE
Multiple Jobs one Trigger ?

### DISCUSSION THREAD

**Discussion by CrazyMoka** [quartznet/quartznet#1652] [Q&A] [answered]
In the docs it mentions you can have multiple triggers for one Job. What about the other way arround?
I need 5 jobs to run at midnight. What is the correct way to handle this with out creating 5 distict triggers for the jobs?

I asked previously, but my over complicated the question.

**Comment by lahma** [comment, marked as answer]
This is how Quartz works. Triggers are the way to define distinct schedules, job's are "just the code that runs". It should be just a `for` loop to create multiple triggers with same configuration, but the power comes from the fact that you can then later adjust the triggers easily to have different characteristics.
